src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +510,36 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
+
+    /**
+     * Lists all the direct forks of this repository, sorted by {@link Sort#NEWEST Sort.NEWEST}
+     */
+    public PagedIterable<GHRepository> listForks() {
+      return listForks(null);
+    }
+
+    /**
+     * Lists all the direct forks of this repository, sorted by the given sort order.
+     * @param sort the sort order. If null, defaults to {@link Sort#NEWEST Sort.NEWEST}.
+     */
+    public PagedIterable<GHRepository> listForks(final Sort sort) {
+        return new PagedIterable<GHRepository>() {
+            public PagedIterator<GHRepository> iterator() {
+                return new PagedIterator<GHRepository>(root.retrieve().asIterator(getApiTailUrl("forks" + ((sort == null)?"":("?sort="+sort.toString().toLowerCase(Locale.ENGLISH)))), GHRepository[].class)) {
+                    @Override
+                    protected void wrapUp(GHRepository[] page) {
+                        for (GHRepository c : page)
```

Comment:
That is your opinion, obviously not that of the original author.

I find it more important to follow existing conventions when making such a small contribution to an existing bigger project, rather than to impose my current opinion about what is good or bad coding style onto it. I consider consistency to be more important than personal taste. (and I completely agree with you that you always should use brackets, but that is besides the point)

I also smell some serious case of bikeshedding here...